frontend/appflowy_flutter/lib/plugins/document/presentation/editor_page.dart

```diff
@@ -80,6 +81,12 @@ class _AppFlowyEditorPageState extends State<AppFlowyEditorPage> {
   late final styleCustomizer = EditorStyleCustomizer(context: context);
   DocumentBloc get documentBloc => context.read<DocumentBloc>();
 
+  @override
+  void initState() {
+    super.initState();
+    commandShortcutEvents.add(showEmojiPickerEvent(context));
```

Comment:
IIRC, if you mark `commandShortcutEvents` as late final then you can use the `BuildContext` within the initialization of `commandShortcutEvents`.

Reply:
As Alex mentioned, you can mark `commandShortcutEvents` as late if this command relies on the build context.